packages/babel-traverse/src/path/family.js

```diff
@@ -67,6 +67,30 @@ export function getSibling(key) {
   });
 }
 
+export function getPrevSibling() {
+  return this.getSibling(this.key - 1);
+}
+
+export function getNextSibling() {
+  return this.getSibling(this.key + 1);
+}
+
+export function forEachNextSibling(callback = Function.prototype) {
```

Comment:
Just like `Array.prototype.forEach` allows a `context` argument, it may be worth including that second argument here too.